web/client/epics/__tests__/playback-test.js

```diff
@@ -315,4 +315,24 @@ describe('playback Epics', () => {
             }
         }, ANIMATION_MOCK_STATE);
     });
+
+    it('switchOffSnapToLayer - skip toggle animation mode when timeline is not visible (collapsed)', done => {
+        const state = {...ANIMATION_MOCK_STATE,
+            timeline: {
+                selectedLayer: 'playback:selected_layer',
+                settings: {
+                    collapsed: true
+                }
+            }};
+        testEpic(addTimeoutEpic(switchOffSnapToLayer, 1000), 1, changeLayerProperties("playback:selected_layer", {visibility: false}), (actions) => {
+            try {
+                expect(actions.length).toBe(2);
+                expect(actions[0].type).toBe(TEST_TIMEOUT);
+                expect(actions[1].type).toBe('EPIC_COMPLETED');
+                done();
+            } catch (e) {
+                done(e);
+            }
+        }, state, false, true);
+    });
```

Comment:
can you add two tests 
when a CHANGE_LAYER_PROPERTIES happens on other layers with others properties different from visibility AND
- timeline is visible 
- timeline is not visible

Reply:
Thanks @MV88, added coverage for these cases